src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -99,6 +100,75 @@ private protected virtual void ChangeVisualState(bool useTransitions)
 		partial void UnregisterSubView();
 		partial void RegisterSubView(View child);
 
+		#region IsEnabled DependencyProperty
+
+		// Note: we keep the event args as a private field for perf consideration: This avoids to create a new instance each time.
+		//		 As it's used only internally it's safe to do so.
+		[ThreadStatic]
+		private static IsEnabledChangedEventArgs _isEnabledChangedEventArgs;
+
+		public event DependencyPropertyChangedEventHandler IsEnabledChanged;
+
+		[GeneratedDependencyProperty(DefaultValue = true, ChangedCallback = true, CoerceCallback = true, Options = FrameworkPropertyMetadataOptions.Inherits)]
+		public static DependencyProperty IsEnabledProperty { get; } = CreateIsEnabledProperty();
+
+		public bool IsEnabled
+		{
+			get => GetIsEnabledValue();
+			set => SetIsEnabledValue(value);
+		}
+
+		private void OnIsEnabledChanged(DependencyPropertyChangedEventArgs args)
+		{
+#if UNO_HAS_MANAGED_POINTERS
+			UpdateHitTest();
+#endif
+
+			_isEnabledChangedEventArgs ??= new IsEnabledChangedEventArgs();
+			_isEnabledChangedEventArgs.SourceEvent = args;
+
+			OnIsEnabledChanged(_isEnabledChangedEventArgs);
+
+#if __ANDROID__
+			this.Enabled = (bool)args.NewValue;
+#elif __IOS__
+			UserInteractionEnabled = (bool)args.NewValue;
+#elif __MACOS__
+			// UserInteractionEnabled = (bool)args.NewValue; // UNO-TODO: Set MacOS native equivalent
+#endif
+
+			IsEnabledChanged?.Invoke(this, args);
+
+			// TODO: move focus elsewhere if control.FocusState != FocusState.Unfocused
```

Comment:
Is there a related task somewhere for this ToDo ? (if so precise the related link)

Reply:
@agneszitte This is an existing code. @MartinZikmund would probably be best to answer this.

Reply:
The `UpdateHitTest` only requests [an update of the coerced "HitTest" property](https://github.com/unoplatform/uno/blob/669a0c304da2634e691cd9b3ed0cd48db59dc1f0/src/Uno.UI/UI/Xaml/UIElement.Pointers.Managed.cs#L37) which ends by [validating the `IsEnable` property using the `IsEnabledOverride()`](https://github.com/unoplatform/uno/blob/669a0c304da2634e691cd9b3ed0cd48db59dc1f0/src/Uno.UI/UI/Xaml/Controls/Control/Control.cs#L61). So as long as this method override has not been removed it should not cause any issue.